### PR TITLE
fix dead link and remove duplicate text

### PR DESCRIPTION
**Pull request by KarolisRam**

### CODE REVIEW

**Comment by cdfreeman-google** [comment]
@frt03 , could you double check this is okay?

File: brax/experimental/composer/README.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ composer.register_env(env_name=env_name, env_desc=env_desc)
 env = composer.create(env_name=env_name)
 ```
 
-Lastly, while less recommended, `desc_edits` can be used for dynamic environment editing by directly modifying `env_desc` dictionary object. For examples, see [envs/ant_descs.py](https://github.com/google/brax/tree/main/brax/experimental/composer/envs/ant_descs.py).
+Lastly, while less recommended, `desc_edits` can be used for dynamic environment editing by directly modifying `env_desc` dictionary object.
```
Comment (frt03) [inline brax/experimental/composer/README.md:62]: You may replace the link as https://github.com/google/brax/blob/main/brax/experimental/braxlines/experiments/composer_sweep.py

**Comment by frt03** [comment]
@KarolisRam
Hi,
I added a short comment. Could you edit it? Thanks!

**Comment by erikfrey** [comment]
Closing this due to inactivity, but feel free to repoen if you'd like to make any changes.

**Pull request closed without merging**